keyboards/kegen/gboy/config.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2022 Matt Chan
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4b65   // Kegen
+#define PRODUCT_ID      0x6762   // G-Boy
+#define DEVICE_VER      0x0001
+#define PRODUCT         G-Boy
```

Comment:
```suggestion
```

Reply:
Why do I need to remove this? If I am adding support for VIA, do I need this? I've noticed other boards in the QMK repo that define vendor, product, device version, and manufacturer. It's not clear why to me why this should be removed.

Reply:
Nevermind, I found the answer on the QMK Discord server. QMK is moving towards a data-driving configuration. Nice.

Reply:
Discussed on discord- link to PR checklist given with data driven docs.